<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLMarkupsDisplayNodeWidget</class>
 <widget class="QWidget" name="qMRMLMarkupsDisplayNodeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Markups Display Node</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="5" column="0">
    <widget class="QLabel" name="glyphScaleLabel">
     <property name="text">
      <string>Glyph Size:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="glyphSizeIsAbsoluteButton">
       <property name="text">
        <string>absolute</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ctkSliderWidget" name="glyphScaleSliderWidget">
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>20.000000000000000</double>
       </property>
       <property name="suffix">
        <string> %</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="qMRMLSliderWidget" name="glyphSizeSliderWidget">
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="pageStep">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="1">
    <widget class="ctkSliderWidget" name="textScaleSliderWidget">
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>20.000000000000000</double>
     </property>
     <property name="suffix">
      <string> %</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="textScaleLabel">
     <property name="text">
      <string>Text Size:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="VisibilityCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="opacityLabel">
       <property name="text">
        <string>Opacity:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ctkSliderWidget" name="opacitySliderWidget">
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="VisibilityLabel">
     <property name="text">
      <string>Visibility:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="ctkCollapsibleGroupBox" name="SliceDisplayCollapsibleGroupBox">
     <property name="title">
      <string>Advanced</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="DisplayNodeViewLabel">
        <property name="text">
         <string>View:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="selectedColorLabel">
        <property name="text">
         <string>Selected Color:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="unselectedColorLabel">
        <property name="text">
         <string>Unselected Color:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ctkColorPickerButton" name="selectedColorPickerButton">
        <property name="displayColorName">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="ctkColorPickerButton" name="unselectedColorPickerButton">
        <property name="displayColorName">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="glyphTypeLabel">
        <property name="text">
         <string>Glyph Type:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="glyphTypeComboBox"/>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="qMRMLMarkupsFiducialProjectionPropertyWidget" name="pointFiducialProjectionWidget"/>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLDisplayNodeViewComboBox" name="DisplayNodeViewComboBox">
        <property name="toolTip">
         <string>Select views in which to show this node. All unchecked shows in all 3D and 2D views.</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="PointLabelsVisibilityLabel">
        <property name="text">
         <string>Point Labels:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="PointLabelsVisibilityCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="ctkCollapsibleGroupBox" name="InteractionGroupBox">
     <property name="title">
      <string>Interaction</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="interactionLabel">
        <property name="text">
         <string>Visible:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="interactionCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLCheckableNodeComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLCheckableNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLDisplayNodeViewComboBox</class>
   <extends>qMRMLCheckableNodeComboBox</extends>
   <header>qMRMLDisplayNodeViewComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLSliderWidget</class>
   <extends>ctkSliderWidget</extends>
   <header>qMRMLSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLMarkupsFiducialProjectionPropertyWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLMarkupsFiducialProjectionPropertyWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkColorPickerButton</class>
   <extends>QPushButton</extends>
   <header>ctkColorPickerButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>glyphSizeIsAbsoluteButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>glyphScaleSliderWidget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>glyphSizeIsAbsoluteButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>glyphSizeSliderWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
